diagnostic_feedback/public/js/src/common.js

```diff
@@ -0,0 +1,83 @@
+function Common(runtime, element, initData) {
+    "use strict";
+
+    //contains common js b/w studio and student view
+
+    var cObj = this,
+    //selectors
+        warningMessage = '.validation-msg',
+        globalMessage = '.diagnostic-feedback .msg',
+        visibleUserAnswer = '.diagnostic-feedback .user-answers:visible',
+        validateErrors = '.diagnostic-feedback .validation-error-message';
+
+    cObj.showGlobalMessage = function (msgObj) {
+        // display message at top right of page
+        var _type = '';
+        var title = '';
+
+        var msg = $(globalMessage, element);
+        msg.removeClass('info-msg success-msg error-msg warning-msg');
+
+        if (msgObj.success) {
+            _type = 'success-msg';
+            title = 'Success! ' + msgObj.msg;
+        } else if (msgObj.warning) {
+            _type = 'warning-msg';
+            title = 'Warning! ' + msgObj.msg;
+        } else {
+            _type = 'error-msg';
+            title = 'Error! ' + msgObj.msg;
+        }
+        msg.addClass(_type);
+        msg.find('h3').html(title);
```

Comment:
Also, the titles are not translated, but I see other parts of this code is using ``gettext``. I'm not sure how the translation will work, but it would be good to be consistent.